contracts/CO30_Felix_Safety.tem_kujari/mission.sqm

```diff
@@ -0,0 +1,8060 @@
+							class Value
+							{
+								class data
+								{
+									singleType="ARRAY";
+									class value
+									{
+										items=4;
+										class Item0
+										{
+											class data
+											{
+												singleType="STRING";
+												value="STAND_U1";
+											};
+										};
+										class Item1
+										{
+											class data
+											{
+												singleType="ARRAY";
+												class value
+												{
+													items=5;
+													class Item0
+													{
+														class data
+														{
+															singleType="STRING";
+															value="hubstandingua_idle1";
+														};
+													};
+													class Item1
+													{
+														class data
+														{
+															singleType="STRING";
+															value="hubstandingua_idle2";
+														};
+													};
+													class Item2
+													{
+														class data
+														{
+															singleType="STRING";
+															value="hubstandingua_idle3";
+														};
+													};
+													class Item3
+													{
+														class data
+														{
+															singleType="STRING";
+															value="hubstandingua_move1";
+														};
+													};
+													class Item4
+													{
+														class data
+														{
+															singleType="STRING";
+															value="hubstandingua_move2";
+														};
+													};
+												};
+											};
+										};
+										class Item2
+										{
+											class data
+											{
+												singleType="BOOL";
+												value=1;
+											};
+										};
+										class Item3
+										{
+											class data
+											{
+												singleType="BOOL";
+												value=0;
+											};
+										};
+									};
+								};
+							};
+						};
+						nAttributes=2;
+					};
+				};
+			};
+			class Attributes
+			{
+			};
+			id=104;
+			atlOffset=9.5367432e-007;
+		};
+		class Item30
+		{
+			dataType="Trigger";
+			position[]={9030.6689,11.501515,6134.5532};
+			class Attributes
+			{
+				name="supplyArrive";
+				condition="((date select 3) >= 8)";
```

Comment:
I would run a test to make sure the dates are actually fixed, hasn't been fully tested yet